app/bt/AffectedUserFlow/verificationAPI.ts

```diff
@@ -0,0 +1,122 @@
+import Config from 'react-native-config';
+
+const baseUrl = Config.GAEN_VERIFY_URL;
+const verifyUrl = `${baseUrl}/api/verify`;
+const certificateUrl = `${baseUrl}/api/certificate`;
+
+const defaultHeaders = {
+  'content-type': 'application/json',
+  accept: 'application/json',
+  'X-API-Key': Config.GAEN_VERIFY_API_TOKEN,
+};
+
+export type Token = string;
+
+interface NetworkSuccess<T> {
+  kind: 'success';
+  body: T;
+}
+interface NetworkFailure<U> {
+  kind: 'failure';
+  error: U;
+}
+
+export type NetworkResponse<T, U = 'Unknown'> =
+  | NetworkSuccess<T>
+  | NetworkFailure<U>;
+
+type CodeVerificationSuccess = VerifiedCodeResponse;
+
+export type CodeVerificationError =
+  | 'InvalidCode'
+  | 'VerificationCodeUsed'
+  | 'Unknown';
+
+interface VerifiedCodeResponse {
+  error: string;
+  testDate: string;
+  testType: string;
+  token: string;
+}
+
+export const postVerificationCode = async (
+  code: string,
+): Promise<NetworkResponse<CodeVerificationSuccess, CodeVerificationError>> => {
+  const url = verifyUrl;
```

Comment:
How do we feel about inlining the url with the fetch call? Or maybe move the assignment closer?

Reply:
works for me! i do like keeping the urls at the top for documentation purposes, but we can inline em.